homodeus_external/xml_folder/dialog_context.xml

```diff
@@ -0,0 +1,226 @@
+            <answer rel_info='affirmative'>fine</answer>
+            <answer rel_info='affirmative'>give blessing</answer>
+            <answer rel_info='affirmative'>gladly</answer>
+            <answer rel_info='affirmative'>indeed</answer>
+            <answer rel_info='affirmative'>k</answer>
+            <answer rel_info='affirmative'>obvioulsy</answer>
+            <answer rel_info='affirmative'>okay</answer>
+            <answer rel_info='affirmative'>ok</answer>
+            <answer rel_info='affirmative'>okie dokie</answer>
+            <answer rel_info='affirmative'>of course</answer>
+            <answer rel_info='affirmative'>permit</answer>
+            <answer rel_info='affirmative'>sure</answer>
+            <answer rel_info='affirmative'>totally</answer>
+            <answer rel_info='affirmative'>yah</answer>
+            <answer rel_info='affirmative'>yeah</answer>
+            <answer rel_info='affirmative'>yes</answer>
+            <answer rel_info='affirmative'>yep</answer>
+            <answer rel_info='affirmative'>you bet</answer>
+            <answer rel_info='affirmative'>yup</answer>
+
+            <answer rel_info='negative'>decline</answer>
+            <answer rel_info='negative'>deny</answer>
+            <answer rel_info='negative'>disagree</answer>
+            <answer rel_info='negative'>disallow</answer>
+            <answer rel_info='negative'>disapprove</answer>
+            <answer rel_info='negative'>forget it</answer>
+            <answer rel_info='negative'>nah</answer>
+            <answer rel_info='negative'>nay</answer>
+            <answer rel_info='negative'>negative</answer>
+            <answer rel_info='negative'>never</answer>
+            <answer rel_info='negative'>no</answer>
+            <answer rel_info='negative'>nope</answer>
+            <answer rel_info='negative'>not</answer>
+            <answer rel_info='negative'>refuse</answer>
+            <answer rel_info='negative'>reject</answer>            
+            <answer rel_info='negative'>turndown</answer>
+        </listen>
+        <feedback>
+            <answer>Alright it's </answer>
+            <answer>I hear you boss, it's </answer>
+            <answer>Noted, it's </answer>
+            <answer>Okay, it's </answer>
+        </feedback>
+        <repeat>
+            <answer>yes or no ?</answer>
+            <answer>Do you agree or not?</answer>
+            <answer>So... I'm not sure, is that an affirmative?</answer>
+        </repeat>
+        <deadline>
+            <answer> I reach the limit of my patience. Have a good day!</answer>
+            <answer> Okay, it seems like I really can not understand what you are saying. Please talk to someone else</answer>
+            <answer> My microphones must have problems, but I can not hear what you are saying, I'll go back to work</answer>
+            <answer> Okay Okay Okay, because I am now shy to make you repeat one more time, I'll just stop this conversation. Goodbye</answer>
```

Comment:
"I am now shy to make you repeat" C'est juste même pas proche d'un syntaxe valide